### DISCUSSION TITLE
How could I show the ET and BT of a specific point in the curve?

### DISCUSSION THREAD

**Discussion by Fayky23** [artisan-roaster-scope/artisan#966] [Q&A]
Artisan could show me the ET and BT of a specific point when I move the mouse cursor to it at version 2.0 (like below), 
<img width="960" alt="微信图片_20220924222655" src="https://user-images.githubusercontent.com/61074833/192103800-a6188c2f-c183-426d-9cf6-d6cfd0c36f71.png">
However, when I updated Artisan to version 2.6, I can't see the same indication (like below), 
![image](https://user-images.githubusercontent.com/61074833/192103843-044f2c8b-ea0b-49bb-9cf5-0aefc029ca5c.png)
so how could I show them again?

**Comment by MAKOMO** [comment]
Press key 'd' on your keyboard to toggle the coordinate widget between TEMP, RoR and OFF mode.

On our YouTube channel you can find a recording of the first Royal Coffee Webinar where our Dave explains this as well as many other things around this coordinate widget: https://www.youtube.com/watch?v=tJKRJrEPeEQ
